### PR TITLE
[fr] COVID_19 modified

### PR DESCRIPTION
**Pull request by GillouLT**
To address: https://github.com/languagetooler-gmbh/languagetool-premium/issues/6017

2 rules modified to avoid suggesting hybrid (only first letter capitalization) _Covid-19_
1 picky rule created for the same hybrid form

Edit 23/11:

Most of the rules from COVID_19 have been moved to style in a new rulegroup COVID_19_GRAPHIE: 

1 “Covid 19” >> “covid 19”
2 “Covid-19” >> “covid-19|COVID-19”
3 “Covid19” >> “COVID-19|covid-19|covid 19”
4 "Covid" >> "covid"

LE_COVID (style) has been modified in order to avoid overlapping, both rules should work in tandem, first 
COVID_19_GRAPHIE will correct the capitalization and, after this change, LE_COVID should suggest the feminine form.
The message from LE_COVID (style) hasn't been changed, it seems well phrased and doesn't take any strong position, which is good.

The priority of LE_COVID has been changed to be lower than COVID_19_GRAPHIE to make sure the tandem system works, and also because COVID_19_GRAPHIE seems more "important" than the gender issue.

### CODE REVIEW

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/style.xml [hunk 1]
```diff
@@ -100,7 +100,7 @@ Pour en savoir plus sur les tone tags, suivez le lien vers notre documentation (
         <rule id="NOUVEL_AN" name="Nouvel An" tone_tags="general">
             <pattern case_sensitive="yes">
                 <token>nouvel</token>
-                <token regexp="yes">an</token>
+                <token>an</token>
```
Comment (GillouLT) [inline languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/style.xml:103]: Maven test warning fixed

**Comment by LucieSteib** [comment]
The Académie recommands not using a dash -_- 
https://www.academie-francaise.fr/le-covid-19-ou-la-covid-19
Larousse goes for ALL caps and dash 
[COVID-19](https://www.larousse.fr/dictionnaires/francais/COVID-19/188582)
Robert goes for dash and probably no caps at all (exept obvi on sentence start)
[Patients ayant le covid-19](https://dictionnaire.lerobert.com/definition/covid)
This is a nightmare. 
Also our blog uses Covid with a capital start in the linking https://languagetool.org/insights/fr/poste/nouveaux-mots-du-dico-2023/#les-mots-nouveaux-li%C3%A9s-%C3%A0-la-pand%C3%A9mie-mondiale.
What do we do? 
We can talk with Anne-So about that on Thursday... I guess we have to make a choice (or let the users do whatever they want, 
Also for the moment spelling.text contains:
```
covid
COVID
Covid
covid-19
COVID-19
Covid-19
```

**Comment by GillouLT** [comment]
True, let's leave the issue opened until we decide this week :)

**Comment by GillouLT** [comment]
Can you have a second look @LucieSteib? I've updated the principal comment of the issue :)

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/style.xml [hunk 2]
```diff
@@ -850,6 +850,61 @@ Pour en savoir plus sur les tone tags, suivez le lien vers notre documentation (
...
+            <rule>
+                <antipattern>
+                    <token postag="SENT_START"/>
+                    <token>Covid</token>
+                </antipattern>
+                <pattern>
+                    <token case_sensitive="yes">Covid</token>
+                    <token>19</token>
+                </pattern>
+                <message>Privilégiez la forme en minuscules sans majuscule initiale, conformément aux graphies acceptées.</message>
```
Comment (LucieSteib) [inline languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/style.xml:864]: praise: oh that's beautiful!

**Review by LucieSteib — APPROVED** [review]
LGTM! :dna:

**Pull request merged**